common/reflect_resource.go

```diff
@@ -254,6 +254,25 @@ func typeToSchema(v reflect.Value, t reflect.Type, path []string) map[string]*sc
 			scm[fieldName].Type = schema.TypeList
 			elem := typeField.Type.Elem()
 			sv := reflect.New(elem).Elem()
+			nestedSchema := typeToSchema(sv, elem, append(path, fieldName, "0"))
+			if strings.Contains(tfTag, "suppress_diff") {
```

Comment:
Where can I see what fields are tagged with suppress_diff for SRTI?

Reply:
This file is not part of the PR, added a comment to make this clear.